src/libs/actions/Policy/Policy.ts

```diff
@@ -342,13 +342,32 @@ function deleteWorkspace(policyID: string, policyName: string) {
 function setWorkspaceAutoReportingFrequency(policyID: string, frequency: ValueOf<typeof CONST.POLICY.AUTO_REPORTING_FREQUENCIES>) {
     const policy = getPolicy(policyID);
 
+    const wasPolicyManuallyReported = PolicyUtils.getCorrectedAutoReportingFrequency(policy) === CONST.POLICY.AUTO_REPORTING_FREQUENCIES.MANUAL;
```

Comment:
Isn't this variable more telling us "wasPolicyOnManualReporting" instead of "was it manually reported"? Like "manually reporting a policy" doesn't really exist, so maybe we can choose a better variable name here.